MARIE_AND_BERT/readme.md

```diff
@@ -0,0 +1,143 @@
+# Marie and BERT (Marie 3.0)
+
+The ``Marie and Bert`` a.k.a `Marie 3.0` project is developed by [Xiaochi Zhou](xz378@cam.ac.uk)
+and [Shaocong Zhang](sz375@cam.ac.uk) and [Mehal Agarwal](ma988@cam.ac.uk).
+
+A demonstration webpage is deployed at [Marie Website](http://159.223.42.53:5003/)
+
+## Architecture
+
+The project contains the following parts:
+```
+├── MARIE_AND_BERT
+│   ├── DATA (for storing all binary files and large files) 
+│   │   ├── CrossGraph 
+│   │   ├── KG
+│   │   │   ├── [ontology names] (for all ontology-specific files) 
+│   │   ├── Dictionaries (for all IRI-label/value mapping files)
+│   │   ├── EntityLinking (for entity linking models)
+│   ├── Marie (scripts for runtime Marie system) 
+│   ├── KGToolbox (scripts for creating dataset) 
+│   ├── Training (scripts for model and embedding training) 
+│   ├── Evaluation (create evaluation dataset and evaluation) 
+│   ├── static (js/css files for frontend)
+│   ├── templates (html files for frontend)
+│   ├── main.py (flask server)
+│   ├── Dockerfile 
+│   ├── requirements.txt 
+```
+
+## Dataset creation 
+The dataset for training the KG embedding and models are mostly script 
+created. To create or amend the dataset, see [readme.md for dataset creation](./KGToolbox/readme.md)
+
+
+## Training
+Multiple models are trained in this system: 
+1. Knowledge Graph Embedding
+2. Relation Prediction
+3. Score Alignment
+4. Entity Linking 
+
+For details on training the models, see [readme.md for embedding and model training](./Training/readme.md) and the [readme.md for entity linking training](./Training/EntityLinking/EL_training.md)
+ 
+  
+## Running 
+
+Install `Python 3.8` and Run `pip install requirements.txt`. It is recommended 
+to use conda for creating a virtual environment. 
+
+### Required files: 
+
+1. Create a `DATA` folder under `MARIE_AND_BERT`
+2. Download CrossGraph.zip from [CrossGraph.zip](http://159.223.42.53:8080/CrossGraph.zip), unzip under `DATA`
```

Comment:
As convenient as it is to just download black-box zip archives, anyone who wants to rebuild the system from scratch let alone develop it further will need to know what's in these files and how to create them. If I had to re-create any of these files, given the current documentation, I would really struggle. Can you please, for each of the zip files mentioned here, add explicit step-by-step instructions what one needs to do to create them?